docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+  mqtt2kasa:
+    # uncomment to build using local Dockerfile
+    # build: 
+      # context: .
+      # dockerfile: Dockerfile
+    image: flavio-fernandes/mqtt2kasa
```

Comment:
As I mentioned, I can update this if the docker hub account would be something else. For now, it's `flavio-fernandes`